docs/DeveloperGuide.adoc

```diff
@@ -400,6 +400,9 @@ _Figure 5.6.1 : Saving documentation as PDF files in Chrome_
 
 There are three ways to run tests.
 
+Note that due to the nature of the Date Borrow field in Person Class, +
+it is advisable that developers do not test at midnight where the date might change.
```

Comment:
What will be the consequences of testing at midnight?

Reply:
When the date changes, it will not match certain tests. This is because the Date Borrow field is created on the day itself.